package.json

```diff
@@ -32,25 +32,23 @@
     "commander": "2.9.0",
     "event-stream": "3.3.2",
     "express": "4.13.4",
-    "lodash": "4.6.1",
     "irc-framework": "1.0.10",
+    "lodash": "4.11.1",
     "mkdirp": "0.5.1",
-    "moment": "2.12.0",
+    "moment": "2.13.0",
     "read": "1.0.7",
-    "request": "2.69.0",
-    "slate-irc": "0.8.1",
+    "request": "2.72.0",
     "socket.io": "1.4.5",
-    "spdy": "3.2.3"
+    "spdy": "3.3.2"
   },
   "devDependencies": {
     "chai": "3.5.0",
-    "eslint": "2.4.0",
-    "grunt": "0.4.5",
-    "grunt-cli": "0.1.13",
```

Comment:
Removed as it's installed by grunt either way. http://gruntjs.com/blog/2016-02-11-grunt-1.0.0-rc1-released#npm-scripts